### PR TITLE
Team motto

### PR DESCRIPTION
**Pull request by SunnyRaj**

### CODE REVIEW

**Comment by nwalters512** [comment]
Can you upload a screenshot of how this looks?

File: android/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Base.Theme.AppCompat.Light.DarkActionBar">
```
Comment (nwalters512) [inline android/src/main/res/values/styles.xml]: I'm curious, why this change? Did the support lib change the namespace?

Comment (SunnyRaj) [inline android/src/main/res/values/styles.xml]: The layout didn't render properly in Android Studio Editor. This was the fix.

**Comment by phil-lopreiato** [comment]
This will depend on https://github.com/the-blue-alliance/the-blue-alliance/pull/1299 in order for data to be shown

**Comment by SunnyRaj** [comment]
Draft I'm working on. Open to comments.

team logo

**Comment by nwalters512** [comment]
Looks solid. I'd like to have an icon for consistency, but I'm not sure if there's a good one to represent the idea of "motto". If not an icon, there should be *something* to indicate that this is the motto, as it might not be immediately clear to an end user.

**Comment by phil-lopreiato** [comment]
I can't think of anything to indicate motto either. I think it'll be alright as is, unless you can think of a good one.

Good stuff @SunnyRaj

**Comment by nwalters512** [comment]
You could try just adding "Motto: " and see how that looks.

**Comment by 1fish2** [comment]
Google Image search found this icon for "motto": http://noahsphotography.com/wp-content/uploads/2014/06/MyMottoIcon2.png

**Comment by phil-lopreiato** [comment]
https://github.com/the-blue-alliance/the-blue-alliance/pull/1299 is merged on the server side, I'm going to merge this for now. That icon doesn't look to material-y and I did some polling of people near me and they all guessed that the field meant team motto. So I think it's good.
